src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/References/SetAttributeCommand.cs

```diff
@@ -1,50 +1,16 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
 
-using System.Linq;
-using System.Threading.Tasks;
 using Microsoft.VisualStudio.ProjectSystem.Properties;
 
 namespace Microsoft.VisualStudio.ProjectSystem.VS.References
 {
-    internal class SetAttributeCommand : IReferenceCommand
+    internal class SetAttributeCommand : BaseSetAttributeCommand
```

Comment:
@sawesonga suggested a better name for this class, and you said it was done, but I don't see it here. Ditto for the "Unset" class.

Reply:
Done. "SetTreatAsUsedAttributeCommand"